src/app/pwa/_components/forms-controls/date-time-range.control.tsx

```diff
@@ -34,17 +106,57 @@ export default function DateTimeRange({ name }: DateTimeRangeProps) {
         }))
     }
 
+    const formatDateTimeForInput = (isoString: string) => {
+        const date = new Date(isoString)
+        return {
+            date: date.toISOString().split('T')[0],
+            time: date.toTimeString().slice(0, 5)
+        }
+    }
+
     return (
-        <>
+        <div className="space-y-4">
             <input type='hidden' name={name} value={JSON.stringify(localValue)} />
-            <div className='flex flex-row items-center justify-between border-b border-[#ebebeb] pb-2 text-black'>
```

Comment:
If I recall correctly these borders and paddings were added to match current UI design, were them removed for some reason or maybe by mistake?

Reply:
I changed them a bit because they wouldn't fit well with the new timezone tab, like overlapping lines with select box